### ISSUE TITLE
SMTP server not working for sending mails 

### ISSUE THREAD

**Issue by deshraj** [Cloud-CV/EvalAI#146]
When a user registers, then a mail needs to be sent to the user's email id. 

The current error is: 

```
SMTPServerDisconnected at /api/auth/registration/
please run connect() first
```

**Comment by RishabhJain2018** [comment]
@deshraj Can I fix this?

**Comment by deshraj** [comment]
Sure. Assigning it to you. :-)

**Comment by aka-jain** [comment]
@deshraj : but email field is not mandatory to fill during registration time, so do we still need it and also is there any type of email verification?

**Comment by deshraj** [comment]
Oh, I didn't realize this. We can make it mandatory just setting a variable in settings. Will add it. 

Thanks for pointing this. :-)

**Comment by aka-jain** [comment]
Also, do we need email verification after sign up?

**Comment by deshraj** [comment]
Yes, we do.

**Comment by RishabhJain2018** [comment]
@deshraj Which EMAIL_HOST we will be using?

**Comment by deshraj** [comment]
@RishabhJain2018 can you just test it on your local machine using any email host such as `smtp.gmail.com`. We will change it once we are sure that it is working.

**Comment by deshraj** [comment]
Resolved in #184 . Closing it.
